opg-elastictrim/Makefile

```diff
@@ -0,0 +1,8 @@
+.PHONY: build
+
+registryUrl ?= registry.service.opg.digital
+image = opg-elastictrim
+
+build:
+	docker build -t "$(registryUrl)/$(image)" .
+	# docker tag  "$(registryUrl)/$(image)" "$(registryUrl)/$(image):$(newtag)"
```

Comment:
No need to retag? (do we want to delete this line?

Reply:
They all look like that, but yes I can remove the commented line.